### ISSUE TITLE
How to use this plugin + minify/urglify output code?

### ISSUE THREAD

**Issue by tangkhaiphuong-colpal** [fossamagna/gas-webpack-plugin#388]
Hi, this plugin is great! But I need output code can uglify/minify?

**Comment by fossamagna** [comment]
@tangkhaiphuong-colpal 
Output written as `Code.gs` in README.md is use case with `none` as webpack  mode. It is example non minify. Please use webpack mode is `production`. You Should be able to got minified script.

```sh
webpack --mode production
```

**Comment by alecmev** [comment]
I'm using Webpack 4, and if I don't set `optimization.minimize` to `false` then the function isn't prepended to the output.

**Comment by VladimirMikulic** [comment]
@fossamagna Setting mode to production gives the same result. It's still not minified.

**Comment by VladimirMikulic** [comment]
The problem seems to be in the file naming. If I set the output file to be `file`.**gs**, Webpack won't minify the file. On the other hand, if I set the output file to be `file`.**js**, Webpack will minify the file, but then `gas-webpack-plugin` doesn't work.

**Comment by VladimirMikulic** [comment]
After a bit of troubleshooting, I realized that Webpack doesn't recognize `.gs` files, so it won't minify them.
The only way we can minify our code is by manually installing UglifyJS Webpack plugin and adding it to the config like this:

`webpack.config.js`
```js
const UglifyJsPlugin = require('uglifyjs-webpack-plugin');

const config = {
  // ...
  optimization: {
    minimizer: [
      new UglifyJsPlugin({
        // Minify JS and GS files
        test: /\.(js|gs)$/i
      })
    ]
  },
  // ...
}

module.exports = config;
```

**Comment by fossamagna** [comment]
@tangkhaiphuong-colpal @alecmev @VladimirMikulic 
Thank you for your report. I had can reproduce the issue.
I will try fix to this issue within several days.

**Comment by fossamagna** [comment]
@tangkhaiphuong-colpal @alecmev @VladimirMikulic
I published fixed version as [v1.0.4](https://www.npmjs.com/package/gas-webpack-plugin/v/1.0.4). Please try to use new version.

@VladimirMikulic 
If you would like to use **gs** as extension name , you need to configure options for terser-webpack-plugin.
See also: https://github.com/webpack-contrib/terser-webpack-plugin#test

**Comment by VladimirMikulic** [comment]
@fossamagna I can confirm that this works! Thank you very much!
